src/cmd-build

```diff
@@ -314,7 +347,16 @@ cat > tmp/meta.json <<EOF
 }
 EOF
 
-echo '{ "images": {} }' > tmp/images.json
+cat > tmp/images.json <<EOF
+{
+  "images": {
+    "ostree": {
+        "path": "${ostree_tarfile_path}",
+        "sha256": "${ostree_tarfile_sha256}"
```

Comment:
i was looking at the other image entries. some of them have the `size:` entry and some don't. Maybe let's add in `size:` here and then also get them added to the other entries in a followup PR?